snowpack/src/config.ts

```diff
@@ -638,6 +642,7 @@ function normalizeConfig(_config: SnowpackUserConfig): SnowpackConfig {
       '`devOptions.out` is now `buildOptions.out`! `devOptions.out` will be deprecated in the next major release.',
     );
   }
+  config.root = path.resolve(cwd, config.root);
```

Comment:
I believe one of your failing tests is because `installOptions.cwd` should be set to root:

```
config.installOptions = config.installOptions || {};
config.installOptions.cwd = config.root;
```

This is needed because `installOptions` are passed to esinstall, which is a seperate tool that needs to be told about the new root. It's okay that we keep using `installOptions.cwd` for now.

Reply:
Fixed in [a8530aa](https://github.com/snowpackjs/snowpack/pull/1928/commits/a8530aa68c317697d2b2e6536b09d5ddcd3c1c23)